test/serializer/additional-functions/arguments.js

```diff
@@ -0,0 +1,16 @@
+// does not contain:x = 5;
+
+function additional1(argument) {
```

Comment:
Apropos my comments above, add and another function with parameter name "argument".